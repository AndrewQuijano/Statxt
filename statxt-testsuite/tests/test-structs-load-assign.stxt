struct types {
	int i;
	char c;
	bool t;
	float f;
	string s;
}

struct types2 {
	int i;
	char c;
	bool t;
	float f;
	string s;
}

int main() {
	
	struct types t1;
	struct types t2;

	t1.i = 1;
	t2.i = t1.i;

	t1.c = 'a';
	t2.c = t1.c;

	t1.t = true;
	t2.t = t1.f;

	t1.f = 3.14;
	t2.f = t1.f;

	t1.s = "hello world!"
	t2.s = t1.s;

	print(t2.i);
	print(t2.c);
	print(t2.t);
	print(t2.f);
	print(t2.s);

	return 0;
}