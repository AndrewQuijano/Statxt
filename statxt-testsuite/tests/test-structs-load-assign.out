1
a
true
3.14
hello world!